### PR TITLE
some update README.md1

### PR DESCRIPTION
**Pull request by StanislavMitrofanov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# test-repo
\ No newline at end of file
+# test-repo
```
Comment (Andrey-Shapovalov) [inline README.md:1]: коммент

Comment (StanislavMitrofanov) [inline README.md:1]: do we really need that line?

**Review by Andrey-Shapovalov — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (StanislavMitrofanov) [reply README.md:1]: here we go again

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# test-repo
```
Comment (StanislavMitrofanov) [inline README.md:1]: anything ??

Comment (StanislavMitrofanov) [inline README.md:1]: ???

Comment (StanislavMitrofanov) [reply README.md:1]: yep

Comment (StanislavMitrofanov) [inline README.md:1]: ЪЪЪЪЪ???

**Pull request closed without merging**